packages/taquito/src/signer/interface.ts

```diff
@@ -8,7 +8,7 @@ export interface Signer {
    * @param magicByte Magic bytes 1 for block, 2 for endorsement, 3 for generic, 5 for the PACK format of michelson
    */
   sign(
-    op: {},
+    op: string,
```

Comment:
this changes `op` type from `object` to `string`

Is that intended?

Reply:
Yes, if we look at the classes that implement the Signer interface, each of them has the type string for this property. It makes sense because `op` is the forged bytes (a string). 
I am not too sure why the interface was defined to `{}`.